recipes/libtins/all/CMakeLists.txt

```
@@ -0,0 +1,7 @@
+cmake_minimum_required(VERSION 3.1)
```

```suggestion
cmake_minimum_required(VERSION 2.8.1)
```

This is the version used in the project